specs/ScrollingControls/ScrollPresenter.md

```diff
@@ -0,0 +1,2462 @@
+
+    Double OffsetDelta { get; };
+    Microsoft.UI.Xaml.Controls.ScrollingScrollOptions Options { get; };
+    Int32 CorrelationId { get; set; };
+}
+
+runtimeclass Microsoft.UI.Xaml.Controls.Primitives.ScrollControllerAddScrollVelocityRequestedEventArgs
+{
+    ScrollControllerAddScrollVelocityRequestedEventArgs(Single offsetVelocity, Windows.Foundation.IReference<Single> inertiaDecayRate);
+
+    Single OffsetVelocity { get; };
+    Windows.Foundation.IReference<Single> InertiaDecayRate { get; };
+    Int32 CorrelationId { get; set; };
+}
+
+runtimeclass Microsoft.UI.Xaml.Controls.Primitives.ScrollControllerInteractionRequestedEventArgs
+{
+    ScrollControllerInteractionRequestedEventArgs(Microsoft.UI.Input.PointerPoint pointerPoint);
+
+    Microsoft.UI.Input.PointerPoint PointerPoint { get; };
+    Boolean Handled { get; set; };
+}
+
+runtimeclass Microsoft.UI.Xaml.Controls.Primitives.ScrollingViewChangeRequestedEventArgs
+{
+    Boolean IsAnimating { get; };
+    Double TargetHorizontalOffset { get; };
+    Double TargetVerticalOffset { get; };
+    Single TargetZoomFactor { get; };
+}
+```
+
+## `ScrollPresenter` class
+
+```csharp
+[contentproperty("Content")]
+unsealed runtimeclass Microsoft.UI.Xaml.Controls.Primittives.ScrollPresenter
+    : Microsoft.UI.Xaml.FrameworkElement,
+      Microsoft.UI.Xaml.Controls.IScrollAnchorProvider
+{
+    ScrollPresenter();
+
+    Microsoft.UI.Xaml.Media.Brush Background { get; set; };
+    Microsoft.UI.Xaml.UIElement Content { get; set; };
+    Microsoft.UI.Composition.CompositionPropertySet ExpressionAnimationSources { get; };
+    Double HorizontalOffset { get; };
+    Double VerticalOffset { get; };
+    Single ZoomFactor { get; };
+    Double ExtentWidth { get; };
+    Double ExtentHeight { get; };
+    Double ViewportWidth { get; };
+    Double ViewportHeight { get; };
+    Double ScrollableWidth { get; };
+    Double ScrollableHeight { get; };
+    Microsoft.UI.Xaml.Controls.ScrollingContentOrientation ContentOrientation { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingChainMode HorizontalScrollChainMode { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingChainMode VerticalScrollChainMode { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingRailMode HorizontalScrollRailMode { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingRailMode VerticalScrollRailMode { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingScrollMode HorizontalScrollMode { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingScrollMode VerticalScrollMode { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingChainMode ZoomChainMode { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingZoomMode ZoomMode { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingInputKinds IgnoredInputKinds { get; set; };
+    Double MinZoomFactor { get; set; };
+    Double MaxZoomFactor { get; set; };
+    Microsoft.UI.Xaml.Controls.ScrollingInteractionState State { get; };
```

Comment:
"State" is too ambiguous, should be either ScrollingInterationState or InteractionState.